Library/Homebrew/cmd/install.rb

```diff
@@ -22,7 +22,10 @@
 #:    If `--cc=`<compiler> is passed, attempt to compile using <compiler>.
 #:    <compiler> should be the name of the compiler's executable, for instance
 #:    `gcc-4.2` for Apple's GCC 4.2, or `gcc-4.9` for a Homebrew-provided GCC
-#:    4.9.
+#:    4.9. In order to use LLVM's clang, use `llvm_clang`. To specify the
```

Comment:
Might be good to reference current versions e.g. `gcc`, `gcc-6` or `gcc-4.9`? Could you also add `Please do not file issues if you encounter errors when using this flag.`

Reply:
Yes for sure, I used those examples because that's what was documented previously, and I wasn't sure if I should change them